assets/sass/protocol/components/logos/_logo.scss

```diff
@@ -2,7 +2,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../../includes/lib';
+@use '../../includes/lib' as *;
+@forward '../../includes/lib';
```

Comment:
Why do we both `@use` and `@forward` lib here?

Reply:
I'll have to look, but this may be something I was testing earlier and can be removed.